controllers/PhotosController.php

```diff
@@ -64,7 +64,7 @@ public function actionUpload($class, $item_id)
                         'message' => Yii::t('easyii', 'Photo uploaded'),
                         'photo' => [
                             'id' => $photo->primaryKey,
-                            'image' => $photo->image,
+                            'image_file' => $photo->image_file,
```

Comment:
im not sure in this change. look here https://github.com/noumo/easyii/blob/master/assets/photos/photos.js#L41